server/migrations/20190911202442-add-password-retries-to-user.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+
+    return queryInterface.addColumn("Users", "passwordRetries", {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      defaultValue: 3
```

Comment:
Maybe 5 is a good amount. We don't want to lock out accounts too often. Thinking about people trying to remember whether they put numbers at the end of their passwords etc.